cmd/client_test.go

```diff
@@ -78,3 +78,14 @@ func TestClient_ShowJobSpec_NotFound(t *testing.T) {
 	assert.NotNil(t, client.ShowJobSpec(c))
 	assert.Empty(t, r.Renders)
 }
+
+func TestClient_CreateJobSpec(t *testing.T) {
+	app, cleanup := cltest.NewApplication()
+	defer cleanup()
+
+	client, _ := cltest.NewClientAndRenderer(app.Store.Config)
+	set := flag.NewFlagSet("create", 0)
+	set.Parse([]string{"badinput"})
+	c := cli.NewContext(nil, set, nil)
+	assert.NotNil(t, client.CreateJobSpec(c))
```

Comment:
Good start with the specs, the flag set is always a bit clumsy.

This checks for bad input, let's get a happy path test in there as well:
```
{"initiators":["web"], "tasks":["noop"]}
```

And let's assert that a job was made:

```golang
assert.Equal(t, len(app.Store.Jobs()), 1)
```

Finally, we can wrap this all up by putting the two cases in one table driven test: https://blog.golang.org/subtests

Chainlink example: https://github.com/smartcontractkit/chainlink/blob/chores/go-report/utils/utils_test.go#L76

Let me know if you have any issues or questions 👍 
- Dimitri